src/layers/ngm-layers-item.ts

```diff
@@ -75,6 +76,16 @@ export class LayerTreeItem extends LitElementI18n {
     this.config.visible = !this.config.visible;
     this.actions.changeVisibility(this.config, this.config.visible);
     this.dispatchEvent(new CustomEvent('layerChanged'));
+    // display visibility hint for subsurface layers
+    if (this.config.visible
+      && [LayerType.tiles3d, LayerType.earthquakes].includes(this.config.type!)) {
+      const sidebar = <any> document.getElementsByTagName('ngm-side-bar')[0];
+      if (sidebar.displayUndergroundHint
+        && !sidebar.viewer?.scene.cameraUnderground) {
+        showSnackbarInfo(i18next.t('lyr_subsurface_hint'));
+        sidebar.displayUndergroundHint = false;
```

Comment:
better to replace it with customEvent or use rxjs or modify and use `layerChanged` event (it handles in side-bar) instead of directly getting the element

Reply:
used custom event in https://github.com/swissgeol/ngm/pull/786/commits/db524154538145c673aa40074b9879a8f272fedd